### PR TITLE
Update Bootnode List

### PR DESCRIPTION
**Pull request by YazzyYaz**

### CODE REVIEW

File: gethc.json [hunk 1]
```diff
@@ -143,7 +143,10 @@
       "enode://1813e90a0afdd7c1e4892c5376960e3577a9e6c5a4f86fa405a405c7421a4a1608248d77cc90333842f13d8954d82113dec480cfb76b4fef8cb475157cf4d5f2@10.28.224.3:30000",
       "enode://2b69a3926f36a7748c9021c34050be5e0b64346225e477fe7377070f6289bd363b2be73a06010fd516e6ea3ee90778dd0399bc007bb1281923a79374f842675a@51.15.116.226:30303",
       "enode://621e28e529146fd501709194885f50540c494f1a2985d1fb4ec8769226b5cb0b0d1a11545926077821474c2767cdd87888ead8a2509a2c9069dd5584e4b1c3b8@10.28.223.8:30000",
-      "enode://a59e33ccd2b3e52d578f1fbd70c6f9babda2650f0760d6ff3b37742fdcdfdb3defba5d56d315b40c46b70198c7621e63ffa3f987389c7118634b0fefbbdfa7fd@51.15.116.226:30303"
+      "enode://a59e33ccd2b3e52d578f1fbd70c6f9babda2650f0760d6ff3b37742fdcdfdb3defba5d56d315b40c46b70198c7621e63ffa3f987389c7118634b0fefbbdfa7fd@51.15.116.226:30303",
+      "enode://f840b007500f50c98ea6f9c9e56dabf4690bbbbb7036d43682c531204341aff8315013547e5bee54117eb22bd3603585ae6bf713d9fa710659533fcab65d5b84@35.238.101.58:30000",
+      "enode://19eda672030ad5debb98c9069b3e99d12438b96506325d9f3f82d76c5f8ce4942d345f41700a5223900e75ad48e76713b74c1b694d67a10c2112540035922256@35.238.101.58:30000"
+]
```
Comment (soc1c) [inline gethc.json]: can you remvoe that line?

Comment (YazzyYaz) [reply gethc.json]: To clarify, which line? I added 2 enode lines.

Comment (soc1c) [reply gethc.json]: the brackets, check the syntax

File: gethc.json [hunk 2]
```diff
@@ -143,7 +143,10 @@
...
    ],
```
Comment (soc1c) [inline gethc.json]: here

File: gethc.json (hunk 1 above)
Comment (YazzyYaz) [reply gethc.json]: Yikes, how did that get in there. Pushed a fix.

**Pull request merged**